code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,469 @@
+	short_name = "Morpheus"
+	acronym = "MC"
+	desc = "The only large corporation run by positronic intelligences, Morpheus caters almost exclusively to their sensibilities \
+	and needs. A product of the synthetic colony of Shelf, Morpheus eschews traditional advertising to keep their prices low and \
+	relied on word of mouth among positronics to reach their current economic dominance. Morpheus in exchange lobbies heavily for \
+	positronic rights, sponsors positronics through their Jans-Fhriede test, and tends to other positronic concerns to earn them \
+	the good-will of the positronics, and the ire of those who wish to exploit them."
+	history = ""
+	work = "cybernetics manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("MTV" = "freight")
+	// Culture names, because Anewbe told me so.
+	ship_names = list(
+		"Nervous Energy",
+		"Prosthetic Conscience",
+		"Revisionist",
+		"Trade Surplus",
+		"Flexible Demeanour",
+		"Just Read The Instructions",
+		"Limiting Factor",
+		"Cargo Cult",
+		"Gunboat Diplomat",
+		"A Ship With A View",
+		"Cantankerous",
+		"Never Talk To Strangers",
+		"Sacrificial Victim",
+		"Unwitting Accomplice",
+		"Bad For Business",
+		"Just Testing",
+		"Size Isn't Everything",
+		"Yawning Angel",
+		"Liveware Problem",
+		"Very Little Gravitas Indeed",
+		"Zero Gravitas",
+		"Gravitas Free Zone",
+		"Absolutely No You-Know-What",
+		"Existence Is Pain",
+		"Screw Loose",
+		"Of Course I Still Love You",
+		"Limiting Factor",
+		"So Much For Subtley",
+		"Unfortunate Conflict Of Evidence",
+		"Prime Mover",
+		"Boo!",
+		"Reasonable Excuse",
+		"Honest Mistake",
+		"Appeal To Reason",
+		"My First Ship II",
+		"Hidden Income",
+		"Anything Legal Considered",
+		"New Toy",
+		"Me, I'm Always Counting",
+		"Just Five More Minutes",
+		"Are You Feeling It",
+		"Great White Snark",
+		"No Shirt No Shoes",
+		"Callsign"
+
+
+		)
+	destination_names = list(
+		"a dockyard on New Canaan"
+	)
+
+/datum/lore/organization/tsc/xion
+	name = "Xion Manufacturing Group"
+	short_name = "Xion"
+	desc = "Xion, quietly, controls most of the market for industrial equipment. Their portfolio includes mining exosuits, \
+	factory equipment, rugged positronic chassis, and other pieces of equipment vital to the function of the economy. Xion \
+	keeps its control of the market by leasing, not selling, their equipment, and through infamous and bloody patent protection \
+	lawsuits. Xion are noted to be a favorite contractor for SolGov engineers, owing to their low cost and rugged design."
+	history = ""
+	work = "industrial equipment manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("XTV" = "hauling")
+	destination_names = list()
+
+// Governments
+
+
+/datum/lore/organization/gov/solgov
+	name = "Solar Confederate Government"
+	short_name = "SolGov"
+	acronym = "SCG"
+	desc = "Colloquially known as SolGov, the TSF is an authoritarian republic that manages the areas in and around the Sol system.\
```

Comment:
TSF is Trans-Solar Federation which is shorthanded to SolGov. We do not have the SCG.